README.md

```diff
@@ -117,6 +118,56 @@ k8s_deploy(
 )
 ```
 
+### Multi-Object Actions
+
+It is common practice in the Kubernetes world to have multiple objects that
+comprise an application.  There are two main ways that we support interacting
+with these kinds of objects.
+
+The first is to simply use a template file that contains your N objects
+delimited with `---`, and omitting `kind="..."`.
+
+The second is through the use of `k8s_objects`, which aggregates N `k8s_object`
+rules:
+```python
+# Note the plurality of "objects" here.
+load("@io_bazel_rules_k8s//k8s:objects.bzl", "k8s_objects")
+
+k8s_objects(
+   name = "deployments",
+   objects = [
+      ":foo-deployment",
```

Comment:
Is there a way to use glob to specify a wildcard on a set of these? It might make a good example.

Reply:
In this context, they are targets, so not really.

You could use a glob to generate the `k8s_deploy` rules:
```python
[k8s_kind(
    name = "some-function-of-input",
    template = f,
) for f in glob(["*.kind.yaml"])]
```

I don't do this in Prow, but I do something similar.